asv_bench/benchmarks/inference.py

```diff
@@ -1,117 +1,114 @@
-from .pandas_vb_common import *
-import pandas as pd
+import numpy as np
+import pandas.util.testing as tm
+import pandas._libs.lib as lib
+from pandas import DataFrame, Series, to_numeric
 
+from .pandas_vb_common import setup  # noqa
 
-class DtypeInfer(object):
-    goal_time = 0.2
 
+class NumericInferOps(object):
     # from GH 7332
+    goal_time = 0.2
+    params = ['int64', 'int32', 'uint32', 'float32', 'float64']
```

Comment:
add uint64. you can add int16, int8, uint16, uint8 as well to cover the bases

Reply:
we may want to define these numeric dtypes elsewhere and import here (so we are consistent across the asv),

and prob for datetimelike as well